linux_docker_resources/Dockerfile

```diff
@@ -94,6 +94,7 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
   python3-flake8-docstrings \
   python3-flake8-import-order \
   python3-flake8-quotes \
+  python3-importlib-metadata \
```

Comment:
Can you please confirm the origin of this new dependency? This is added because Debian has split this module from setuptools into its own deb package correct?

Reply:
Yes, it has been split out.

To be perfectly frank, I'm not 100% sure that this is needed.  However, we currently install this in RHEL, and we also install it via pip (conditionally as of this PR).  So I thought it would be best if we were consistent on how we install it.  In the future we may want to consider removing it completely, but I think we would have to test that out separately.